debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -305,86 +305,96 @@ public void shouldConsumeMessagesFromSnapshot() throws Exception {
+                            + "INSERT INTO changepk.test_table VALUES(3, 'newpkcol', 8)" );
+            records = consumeRecordsByTopic( 1 );
 
-        insertRecord = records.recordsForTopic(topicName).get(0);
-        assertEquals(topicName, insertRecord.topic());
-        VerifyRecord.isValidInsert(insertRecord, newPkField, 3);
-        VerifyRecord.isValidInsert(insertRecord, "pk2", 8);
+            insertRecord = records.recordsForTopic( topicName ).get( 0 );
+            assertEquals( topicName, insertRecord.topic() );
+            VerifyRecord.isValidInsert( insertRecord, newPkField, 3 );
+            VerifyRecord.isValidInsert( insertRecord, "pk2", 8 );
 
-        stopConnector();
+            stopConnector();
 
-        // De-synchronize JDBC PK info and decoded event schema
-        TestHelper.execute("INSERT INTO changepk.test_table VALUES(4, 'newpkcol', 20)");
-        TestHelper.execute(
-                "ALTER TABLE changepk.test_table DROP CONSTRAINT test_table_pkey;"
-              + "ALTER TABLE changepk.test_table DROP COLUMN pk2;"
-              + "ALTER TABLE changepk.test_table ADD COLUMN pk3 SERIAL;"
-              + "ALTER TABLE changepk.test_table ADD PRIMARY KEY(newpk,pk3);"
-              + "INSERT INTO changepk.test_table VALUES(5, 'dropandaddpkcol',10)");
-
-        start(PostgresConnector.class, config.getConfig());
-
-        records = consumeRecordsByTopic(2);
-
-        insertRecord = records.recordsForTopic(topicName).get(0);
-        assertEquals(topicName, insertRecord.topic());
-        VerifyRecord.isValidInsert(insertRecord, newPkField, 4);
-        Struct key = (Struct) insertRecord.key();
-        // The problematic record PK info is temporarily desynced
-        assertThat(key.schema().field("pk2")).isNull();
-        assertThat(key.schema().field("pk3")).isNull();
-
-        insertRecord = records.recordsForTopic(topicName).get(1);
-        assertEquals(topicName, insertRecord.topic());
-        VerifyRecord.isValidInsert(insertRecord, newPkField, 5);
-        VerifyRecord.isValidInsert(insertRecord, "pk3", 10);
-        key = (Struct) insertRecord.key();
-        assertThat(key.schema().field("pk2")).isNull();
+            // De-synchronize JDBC PK info and decoded event schema
+            TestHelper.execute( "INSERT INTO changepk.test_table VALUES(4, 'newpkcol', 20)" );
+            TestHelper.execute(
+                    "ALTER TABLE changepk.test_table DROP CONSTRAINT test_table_pkey;"
+                            + "ALTER TABLE changepk.test_table DROP COLUMN pk2;"
+                            + "ALTER TABLE changepk.test_table ADD COLUMN pk3 SERIAL;"
+                            + "ALTER TABLE changepk.test_table ADD PRIMARY KEY(newpk,pk3);"
+                            + "INSERT INTO changepk.test_table VALUES(5, 'dropandaddpkcol',10)" );
+
+            start( PostgresConnector.class, config.getConfig() );
+
+            records = consumeRecordsByTopic( 2 );
```

Comment:
Minor nit: could you revert those whitespace changes? Specifically, we don't use the whitespace around parameter arguments. It looks like you applied the Hibernate template perhaps? We ought to expand CheckStyle for that, too.